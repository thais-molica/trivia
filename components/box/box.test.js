import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Box from "./index";

it("Box renders correctly", () => {
  const tree = renderer.create(<Box>teste</Box>).toJSON();
  expect(tree).toMatchSnapshot();
});
