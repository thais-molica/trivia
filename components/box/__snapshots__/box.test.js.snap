// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders correctly 1`] = `
<div
  className="jsx-2247176598 box"
  onClick={[Function]}
>
  teste
</div>
`;
